dm/tests/dmctl_command/run.sh

```diff
@@ -81,7 +81,20 @@ function run() {
 	cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
 	cp $cur/conf/source2.yaml $WORK_DIR/source2.yaml
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker1/relay_log" $WORK_DIR/source1.yaml
-	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
+	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1 -w $WORKER1_NAME
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"list-member --name worker1" \
+		"\"stage\": \"bound\"" 1 \
+		"\"source\": \"mysql-replica-01\"" 1
+	dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2 -w $WORKER1_NAME
```

Comment:
CI failed,

```
[2022-01-27T03:23:23.958Z] dmctl test cmd: "operate-source create /tmp/dm_test/dmctl_command/source2.yaml"
[2022-01-27T03:23:25.335Z] dmctl test cmd: "operate-source create /tmp/dm_test/dmctl_command/source2.yaml -w wrong-worker"
[2022-01-27T03:23:25.335Z] dmctl test cmd: "operate-source create /tmp/dm_test/dmctl_command/source2.yaml -w worker1"
[2022-01-27T03:23:25.595Z] dmctl test cmd: "operate-source create /tmp/dm_test/dmctl_command/source2.yaml"
[2022-01-27T03:23:25.595Z] command: operate-source create /tmp/dm_test/dmctl_command/source2.yaml "result": true count: 0 != expected: 2
[2022-01-27T03:23:25.595Z] {
[2022-01-27T03:23:25.595Z]     "result": false,
[2022-01-27T03:23:25.595Z]     "msg": "[code=46007:class=scheduler:scope=internal:level=medium], Message: source config with ID mysql-replica-02 already exists",
[2022-01-27T03:23:25.595Z]     "sources": [
[2022-01-27T03:23:25.595Z]     ]
[2022-01-27T03:23:25.595Z] }
```

for line 89, I think create source2 on worker1 should failed because worker1 is bound to source1, but the script didn't detect the fail. Please fix it